eventarc/generic/main.go

```diff
@@ -36,12 +36,15 @@ func GenericHandler(w http.ResponseWriter, r *http.Request) {
 	logAndRespond(w, "Event received!")
 
 	// Log all headers besides authorization header
+	// Assumes headers don't have duplicate keys
```

Comment:
Why? `strings.Join(v, ",")` can join the values into a single string. Alternatively, keep it as a `[]string`.

Reply:
I suppose I haven't seen multiple non-unique headers in a HTTP request seen here.
Changed to the suggestion, thanks.